src/System.Private.CoreLib/shared/System/Activator.cs

```diff
@@ -0,0 +1,62 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Globalization;
+using System.Reflection;
+using System.Runtime.Remoting;
+
+namespace System
+{
+    /// <summary>
+    /// Activator contains the Activation (CreateInstance/New) methods for late bound support.
+    /// </summary>
+    public static partial class Activator
+    {
+        private const BindingFlags ConstructorDefault = BindingFlags.Instance | BindingFlags.Public | BindingFlags.CreateInstance;
+
+        public static object CreateInstance(Type type, BindingFlags bindingAttr, Binder binder, object[] args, CultureInfo culture) =>
```

Comment:
These methods have DebuggerHidden/DebuggerStepThrough attributes in ProjectN. I think we have to keep those for ProjectN debugging.